How-to-access-Oppia-webpages.md

```diff
@@ -87,9 +88,21 @@ As you go through https://www.oppia.org/, you will find these entities.
 ### Other terms
 
 1. **Collection**: A group of explorations.
-2. **Community**: Anyone who teaches, learns, and contributes to any content on Oppia.
+2. - **Community** Anyone who teaches, learns, and contributes to any content on Oppia. Community members are respectful citizens whose main objective is to steward enriching learning opportunities.
```

Comment:
Please drop the "- " at the beginning.

Though actually I think we can drop this one. I'm not really sure that "community" needs a definition.

Reply:
Dropped the community term.